### ISSUE TITLE
Import fails: AttributeError: 'NoneType' object has no attribute 'reference_skel'

### ISSUE THREAD

**Issue by praecipitator** [BadDogSkyrim/PyNifly#195]
PyNifly version: V13.6.0
Blender version: 3.6 and 4.0.2
Nif was intended for Fallout4

Example nif:
[AAA.zip](https://github.com/BadDogSkyrim/PyNifly/files/13799415/AAA.zip)
I deleted everything from it, and it still won't import.

Made it by SCOLing something in the CreationKit initially, hacked it a bit using outfitstudio, but this happens if I try to import the SCOLed nif straight out of the CK, too. 
Tried taking a vanilla nif from Fallout4, and copying my stuff into it. It seems just doing that irreversibly contaminates the file.

**Comment by BadDogSkyrim** [comment]
Dood, that nif's got nothing in it, at least so far as NifSkope is concerned. You sure you did it right?

**Comment by praecipitator** [comment]
Yes, I tried erasing everything from it until I find the culprit, and ended up with this. I thought such a stripped-down nif would help more, but if you want the full one, here:
[FrontCarBase.zip](https://github.com/BadDogSkyrim/PyNifly/files/13799890/FrontCarBase.zip)

**Comment by BadDogSkyrim** [comment]
OK, this is stupid. Lowercase the file extension and it will import.

**Comment by praecipitator** [comment]
LMAO srsly? 
Okay. Too late for me, I rebuilt the thing from scratch in NifSkope, but, good to know for the next one, thanks!

**Comment by BadDogSkyrim** [comment]
Will be fixed in next release.
